CHANGELOG.asciidoc

```diff
@@ -119,6 +119,9 @@ https://github.com/elastic/beats/compare/v6.4.0...6.x[Check the HEAD diff]
 - Add tag "multiline" to "log.flags" if event consists of multiple lines. {pull}7997[7997]
 - Release `docker` input as GA. {pull}8328[8328]
 - Keep unparsed user agent information in user_agent.original. {pull}7823[7832]
+- Keep unparsed user agent information in user_agent.original. {pull}8537[8537]
+- Better tracking of number of open file descriptors. {pull}7986[7986]
+- Added default and TCP parsing formats to HAproxy module {issue}8311[8311] {pull}8637[8637]
```

Comment:
Unrelated entries

Reply:
Should I delete them and push a new commit?

Reply:
Yes, delete them please.